app/browser/api/ledger.js

```diff
@@ -2445,6 +2444,11 @@ const transitionWalletToBat = () => {
     return
   }
 
+  if (!client) {
+    console.log('client error')
```

Comment:
is this intentional? could be a `console.error` if so

Reply:
Yes it's intentional. Because this is not really an error, it's just log that there was race condition and it' will be triggered again after another call